### PR TITLE
Modifying Readme.md with proper git commands

### LINKED ISSUE #58: Update ReadMe with proper git commands
The README of this repo should be updated with proper git commands to give a headstart to Newcomers.
Would You like to work on this issue?
yes

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #58 
Readme is updated with proper git commands as discussed.

### CODE REVIEW

**Comment by SIMRAN1** [comment]
@imjog @chashmeetsingh please review the pr!

File: README.md [hunk 1]
```diff
@@ -1,26 +1,90 @@
...
+4. Build the project (⌘+B) and check for any errors.
+
+5. Run the app (⌘+R).and test it.
 # Contribute!
 
 > You're now ready to contribute!
 
 Before writing any code, We will highly recommend to have a look at the [Contribution Guidelines](CONTRIBUTING.md) to avoid any unnecessary conflicts.
 
+## Git commands for Configuring remotes
```
Comment (jogendra) [inline README.md]: I think it is not a good idea to include all the git related instructions on Readme file.
- Make a **doc** folder. (As we have in SUSI iOS)
- create `gitconfiguraion.md` file in the doc folder.
- Put this section into gitconfiguration.md file.
- Give link to gitconfiguration in Readme.

**Review by jogendra — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SIMRAN1) [reply README.md]: okk @imjog !

**Comment by SIMRAN1** [comment]
@imjog I have done the changes  as mentioned.

**Review by jogendra — APPROVED** [review]
Looking good to me. Nice work!

**Comment by SIMRAN1** [comment]
Thank u so much @imjog !!

**Pull request merged**